lib/glific/registrations/registration.ex

```diff
@@ -0,0 +1,76 @@
+defmodule Glific.Registrations.Registration do
+  @moduledoc """
+  Registrations are the application form filled by users
+  """
+  use Ecto.Schema
+  import Ecto.Changeset
+  import Ecto.Query, warn: false
+
+  alias Glific.Partners.Organization
+  alias __MODULE__
+
+  # define all the required fields for organization
+  @required_fields [
+    :organization_id
+  ]
+
+  # define all the optional fields for organization
+  @optional_fields [
+    :org_details,
+    :platform_details,
+    :billing_frequency,
+    :finance_poc,
+    :submitter,
+    :signing_authority,
+    :has_submitted,
+    :has_confirmed,
+    :organization_id
```

Comment:
why `organization_id` both in optional and required. Also there should be few more entries in required which are added in the first API

Reply:
> Also there should be few more entries in required which are added in the first API
Those are added as a map in the `org_details` key